spec/index.bs

```diff
@@ -778,6 +798,10 @@ Rendering operations use the following coordinate systems:
     * 0 &leq; v &leq; 1.0
```

Comment:
Made some changes, but I'm not sure if we really want to break it out into different sections per dimension. Take a look and see what you think of the changes.

Reply:
LGTM
And maybe it doesn't really matter to explain majorness since the point is just that the first texel is at (0,0,0) and not (0,1,0) or something. We already have lots of text about bytesPerRow/rowsPerImage anyway.